utils/ifupdown.sh.in

```diff
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/bin/bash
```

Comment:
I can see if I can come up with a POSIX way to do this ... However, note that this script probably would not be used on an embedded system.  This script is intended to be used by ifupdown, which is a Debian/Ubuntu specific package, and bash is considered a required package on Debian/Ubuntu, so it should always be present on any system with ifupdown.  I did add new mstp_config_bridge and ifquery scripts in a separate commit to allow this script to be used on non-Debian/Ubuntu systems, but mstp_config_bridge also requires bash, and ifquery requires python, so again they are unlikely to be used on embedded systems.